MatrixKit/Models/Account/MXKAccountManager.m

```diff
@@ -344,7 +353,23 @@ - (BOOL)isAPNSAvailable
 
 - (void)loadAccounts
 {
-    NSData *accountData = [[NSUserDefaults standardUserDefaults] objectForKey:@"accounts"];
+    NSUserDefaults *sharedDefaults = [MXKAppSettings standardAppSettings].sharedUserDefaults;
+    
+    //Migration of accountData from [NSUserDefaults standardUserDefaults] to sharedDefaults (shared between apps and extensions)
+    NSData *oldAccountData = [[NSUserDefaults standardUserDefaults] objectForKey:kMXKAccountsKey];
+    if (oldAccountData)
+    {
+        [sharedDefaults setObject:oldAccountData forKey:kMXKAccountsKey];
+        [[NSUserDefaults standardUserDefaults] removeObjectForKey:kMXKAccountsKey];
```

Comment:
Before merging, can we temporary comment ```[[NSUserDefaults standardUserDefaults] removeObjectForKey:kMXKAccountsKey];``` out?
The reason is that the user (or the developer as a user) loses his credentials when coming back to an old app version - which is easy when switching between app store, TF and develop versions.

So, please comment the line (and the corresponding one with the synchronize) with a todo comment like: ```// TODO: Erase old location of accountData when the app goes to the app store```